services/movie.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import annotations
```

Comment:
im not sure you need this import here

Reply:
dont know why, but list[int] type in get movies doent work without it, so probably i will leave this one here.